lib/libutils/isoc/include/sys/_default_fcntl.h

```diff
@@ -0,0 +1,74 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
We need a copyright here also. From newlib repository, it seems to me the one that applies here is:
`Copyright (c) 1994-2009  Red Hat, Inc. All rights reserved.`

Reply:
Added in https://github.com/OP-TEE/optee_os/pull/6700/commits/69a53ff771d8ff4badb85a558f91212f1e85d8fc